probe-rs/src/probe/espusbjtag/mod.rs

```diff
@@ -484,7 +484,7 @@ impl DebugProbe for EspUsbJtag {
         let taps = self.scan()?;
         tracing::info!("Found {} TAPs on reset scan", taps.len());
 
-        let selected = 0;
+        let selected = 1;
```

Comment:
This is esp32p4 specific, the HP core(s) debug module is second the in the chain, not first.

Reply:
I've "worked around" this for now, and opened up https://github.com/probe-rs/probe-rs/issues/2058 to discuss how we should handle this sort of thing in the future.